apps/www/components/Article/Page.js

```diff
@@ -311,6 +312,8 @@ const ArticlePage = ({
   const routerQuery = router.query
   const isClimate = !!article?.content?.meta?.climate
 
+  useProlitterisTracking(documentId, cleanedPath)
```

Comment:
`documentId` not stable, will change whenever new version is published.

`repoId` is stable.

One would have to check if `uid` accepts `repoId` characters.

Reply:
good catch! 

Info from prolitteris

> In Verlagen werden die Artikel oftmals mit eindeutigen, internen Schlüsseln versehen. Diese internen Schlüssel können ebenfalls als Zählmarke verwendet werden.
> Dabei gibt es folgendes zu beachten:
> - Ein verlagsinterner Schlüssel setzt sich aus dem Präfix „vzm.<Mitgliedsnummer>-“ und einem beliebigen Schlüssel zusammen. Der Platzhalter „<Mitgliedsnummer>“ muss mit der ProLitteris Mitgliedsnummer des Verlags ersetzt werden.
> - Der Schlüssel darf keine Sonderzeichen enthalten (d.h. nur [a-z][A-Z][0-9] sowie den Bindestrich).
> - Der Schlüssel muss innerhalb des Verlags eindeutig sein.
> - DOI (http://www.doi.org) können ebenfalls als verlagsinterne Schlüssel zum Einsatz kommen. Der Slash "/" nach der Verlagskennung muss durch einen Bindestrich ("-") ersetzt werden. Im DOI dürfen keine weiteren Slashes oder Sonderzeichen vorkommen.
> Beispiel: vzm.970-123456789

Reply:
welche Form hat denn die repoID?

Reply:
wenn es nicht zwingend notwendig ist, würd ich die repoID nicht umwandeln. Wenn ich z.B. überprüfen muss, ob eine zählung funktioniert, kann ich nicht ohne weiteres die repoID nehmen und bei prolitteris suchen, ob diese repoID existiert.

Reply:
Repo-ID sieht etwa so aus: `republik/artikel-zu-irgend-einem-thema`

Laut Specs lese ich, dass keine Slashes erlaubt sind, und wohl auch keine dashes nach der Mitgliedsnummer? Daher Hex-String wohl das schnellsten, einfachste.

Reply:
Weil es das Nachverfolgen und Verifizieren des Trackings erschwert wenn an Prolitteris andere ID's geschickt werden als wir intern für unsere Beiträge verwenden. Wenn ich wissen will, ob Artikel XYZ richtig getrackt wird oder ob die richtigen Autorinnen erfasst wurden, muss ich XYZ.hex() statt eines einfachen substrings unserer bestehenden ID. Von mir aus können wir auch - wie das beispiel oben mit dem DOI - "Den Slash "/" nach der Verlagskennung (republik) mit einem Bindestrich ("-") ersetzten". Das wäre eigentlich am Besten, da von Prolitteris sogar explizit vorgeschlagen.

Reply:
`republik/article-mobilfunk-1 -> republik-article-mobilfunk-1`

Reply:
Fyi, wir verwendeten für die test-implementierung bisher `PROLITTERIS_DEV_UID=republik-test-uid  ` wir wissen also, dass die repoId mit Bindestrich statt slash funktioniert.